app/templates/posts/index.hbs

```diff
@@ -1,11 +1,11 @@
 <h3>All posts</h3>
 
 {{#each model as |post|}}
-  <article class="listing">
-    <h3>{{#link-to "posts.show" post class=post.id}}{{post.title}}{{/link-to}}</h3>
-    <div class="detail body">
-      <span>Post:</span> {{post.body}}
+  {{#link-to "posts.show" post class="post.id uk-link-reset"}}
+    <div class="listing uk-card uk-card-body uk-card-hover">
```

Comment:
Why do you need the class `listing` here? Also, you should use the `{{uk-card}}` component provided by `ember-uikit` (https://adfinis-sygroup.github.io/ember-uikit/#/components/card)